yshin updated api

yshin opened the pull request:
Updated API Docs

areeba-a-abid committed:
Update api.md

Default end_ts in 3.2 changed to 365 days after API call date.

areeba-a-abid committed:
Update api.md

areeba-a-abid committed:
Update api.md

changed default end_ts for rate api back to 371

areeba-a-abid committed:
Update api.md

Updated rate output format and added example output.

areeba-a-abid committed:
Update api.md

areeba-a-abid committed:
Update api.md

yshin committed:
Update api.md

yshin committed:
Update api.md

yshin committed:
wip

yshin committed:
wip

yshin committed:
wip

yshin committed:
wip

yshin committed:
wip

yshin committed:
wip

yshin committed:
wip

yshin committed:
Update api.md

yshin committed:
Update api.md

wlowry88 commented on data-formats/api.md:
```diff
@@ -152,37 +180,129 @@ This API returns __daily__ base price per car. The time granularity is daily, wh
 
 ##### Returns
 
 If prediction is available for this car, base price API will return the following JSON object :
```
base or rates?

wlowry88 commented on data-formats/api.md:
```diff
@@ -152,37 +180,129 @@ This API returns __daily__ base price per car. The time granularity is daily, wh
 import requests
 request = requests.get(
-    'https://api.pfpr.co/v1.0/base?location_code=SFO&car_class=ICAR&start_ts=20160601T014639Z&end_ts=20160602T014639Z',
+    'https://api.pfpr.co/v1.0/base?location_code=SFO&car_class=ICAR&start_ts=20190719T060100Z&end_ts=20200720T060100Z,
```
base or rates?

wlowry88 reviewed (commented):
the url in the docs says rates but the code example says base price?

yshin committed:
Update api.md

yshin committed:
Update api.md

yshin committed:
Update api.md

yshin committed:
Update api.md

yshin committed:
Update api.md

yshin committed:
Update api.md

yshin committed:
Update api.md

wlowry88 committed:
Update api.md

small typos

wlowry88 committed:
Update api.md

wlowry88 committed:
Update api.md

wlowry88 committed:
domain update

we can have guaranteed https on perfectprice.com, it's less confusing to them to have the same domain, lets use use load balancers.

yshin committed:
Update api.md
